### PR TITLE
feat: added a `verbatim_popup` module

### LINKED ISSUE #30: Make a show_rcode module
This module will be a alternative to `get_rcode` which is supposed
- ui button with "Show R code" label
- srv element which shows a popup window containing code. `show_rcode_srv` should work when reactive or static `expression` is passed or `character` vector. Should allow to pass `character` vector with multiple values - module should concatenate/format the code

### PR DESCRIPTION
**Pull request by kpagacz**
Closes #30

Test with the example replacing the contents of the modal. Also, have a look at the tests for examples of what should be allowed and what should not be allowed into the modal.

The idea is that behaviour is the same as the previous `get_rcode` module but without the `get_eval_details` part.

This just shows the content formatted as verbatim. Be aware that it will not format properly without `shiny`s bundled CSS, so it will not look as intended when the UI function is called outside of a `shiny` object (like a fluid page in the example).

I replaced the bloated `clipboard.js` with our own, custom js. Should be short enough to maintain.

I would be glad if someone with MacOS tested the copying function because I have read about some edge cases with the Navigator API and MacOS. I have put some fallback code inspired by an so post, but I would sleep better if it was tested.

### CODE REVIEW

**Review by gogonzo — COMMENTED** [review]
Wundabar. Please add lifecycle experimental badges and add pkgdown entries.

File: NAMESPACE [hunk 1]
```diff
@@ -20,6 +20,8 @@ export(standard_layout)
 export(table_with_settings_srv)
 export(table_with_settings_ui)
 export(updateOptionalSelectInput)
+export(verbatim_popup_srv)
+export(verbatim_popup_ui)
```
[the remark this answers is not in the document]
Comment (pawelru) [reply NAMESPACE:24]: I am just curious why pkgdown workflow does not complain. That should be catched automatically. Will you be able to have a look at this and create a task for IDR if necessary? Many thanks!

[the remark this answers is not in the document]
Comment (kpagacz) [reply NAMESPACE:24]: ? It is there, isn't it?
image

[the remark this answers is not in the document]
Comment (pawelru) [reply NAMESPACE:24]: brain fart - sorry

**Comment by kpagacz** [comment]
Badges added. Pkgdown unchanged, it already lists `verbatim_output` which is the name of the roxygen topic.

**Review by gogonzo — APPROVED** [review]
🚀

**Pull request merged**